resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -0,0 +1,1448 @@
+- slug: general
+  name: General descriptor schema
+  description: General descriptor schema.
+  version: 1.0.0
+  schema:
+    - name: general
+      label: General
+      group:
+        - name: name
+          label: Sample name
+          type: basic:string
+          required: false
+        - name: slug
+          label: Sample slug
+          type: basic:string
+          required: false
+        - name: id
+          label: Sample id
+          type: basic:integer
+          required: false
+        - name: collection
+          label: Collection
+          type: basic:string
+          required: false
+        - name: species
+          label: Species
+          type: basic:string
+          default: Homo sapiens
```

Comment:
It wrong to set the default value to `Homo sapiens`. If this schema is to be the default for all new samples, all non-human samples will be wrongfully annotated by default.